### PR TITLE
Feature/editable operator name

### PR DESCRIPTION
**Pull request by fberanizo**
Adds new column "name" to models.Operator and schemas.Operator …
This column allows the edition of an operator in the interface.
In the database null values are allowed, however the API returns
the task.name in this case.

Updates Swagger docs with new column operator.name …
POST and PATCH operators allow this new column.
All operator responses also have it set.

Enables unit tests on Github Actions …
Runs only test_templates.py while we are testing it.

Fix a lot of errors in tests …
Adds operator.name, operator.status, operator.status_message to test files

Replaces MinIO imagem in Github actions …
Uses one that doesn't need a CMD.

### CODE REVIEW

File: tests/test_tasks.py [hunk 1]
```diff
@@ -36,8 +36,8 @@
 SAMPLE_NOTEBOOK = '{"cells":[{"cell_type":"code","execution_count":null,"metadata":{"tags":["parameters"]},"outputs":[],"source":["shuffle = True #@param {type: \\"boolean\\"}"]}],"metadata":{"kernelspec":{"display_name":"Python 3","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.6.9"}},"nbformat":4,"nbformat_minor":4}'
 
 TASK_ID_2 = str(uuid_alpha())
-EXPERIMENT_NOTEBOOK_PATH_2 = f"minio://{BUCKET_NAME}/tasks/{TASK_ID_2}/Experiment.ipynb"
-DEPLOYMENT_NOTEBOOK_PATH_2 = f"minio://{BUCKET_NAME}/tasks/{TASK_ID_2}/Deployment.ipynb"
+EXPERIMENT_NOTEBOOK_PATH_2 = "Experiment.ipynb"
+DEPLOYMENT_NOTEBOOK_PATH_2 = "Deployment.ipynb"
```
Comment (miguelfferraz) [inline tests/test_tasks.py:40]: Tem outras duas constantes com o mesmo valor declaradas acima, acho que pode remover essas daqui.

**Review by miguelfferraz — COMMENTED** [review]
Muito bom!

**Comment by fberanizo** [comment]
@miguelfferraz fazendo essa tarefa percebi umas coisas:
- PATCH com campos inexistentes no schema não dão erro no fastapi (os campos são apenas ignorados).
- O PATCH de operators/:id/parameters está retornando o operator todo. Acho que era pra retornar só o parâmetro. Entra na fila de ajustes.

**Review by miguelfferraz — APPROVED** [review]
LGTM

**Pull request merged**